WcaOnRails/db/migrate/20180729000001_add_team_for_banned_competitors.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddHiddenTeamForBannedCompetitors < ActiveRecord::Migration[5.1]
+  def change
+    execute <<-SQL
+      INSERT INTO teams (friendly_id, email, rank, created_at, updated_at, hidden) values ('banned', 'disciplinary@worldcubeassociation.org', 90, NOW(), NOW(), TRUE);
```

Comment:
nit: I think it would be better for this to be just like in your seeds file:

```
Team.create(friendly_id: 'banned', rank: 90, email: "disciplinary@worldcubeassociation.org", hidden: true)
```